examples/hypervisor_cluster_profile.go

```diff
@@ -100,11 +104,21 @@ func main() {
 		fmt.Println("#----------------HypervisorClusterProfile updated---------------#")
 	}
 
-	err = ovc.DeleteHypervisorClusterProfile(id)
-	if err != nil {
-		fmt.Println("HypervisorClusterProfile Delete Failed: ", err)
+	//Delete function accepts 2 optional arguments - softDelete(boolean) and force(boolean) till API1200
+	//softDelete is mandatory argument for delete function in API 1600
+	if ovc.APIVersion > 1200 {
```

Comment:
I think we can cteate a block without if,else since softDelete is optional in api1200

Reply:
after having discussion with Nabhajit, came to a point that if we are able to support unified structure in Terraform, then we will go ahead and implement in Go.